src/bower_components/emby-apiclient/apiclient.js

```diff
@@ -193,7 +193,7 @@ define(["events", "appStorage"], function(events, appStorage) {
 
     function getRemoteImagePrefix(instance, options) {
         var urlPrefix;
-        return options.artist ? (urlPrefix = "Artists/" + instance.encodeName(options.artist), delete options.artist) : options.person ? (urlPrefix = "Persons/" + instance.encodeName(options.person), delete options.person) : options.genre ? (urlPrefix = "Genres/" + instance.encodeName(options.genre), delete options.genre) : options.musicGenre ? (urlPrefix = "MusicGenres/" + instance.encodeName(options.musicGenre), delete options.musicGenre) : options.gameGenre ? (urlPrefix = "GameGenres/" + instance.encodeName(options.gameGenre), delete options.gameGenre) : options.studio ? (urlPrefix = "Studios/" + instance.encodeName(options.studio), delete options.studio) : (urlPrefix = "Items/" + options.itemId, delete options.itemId), urlPrefix
+        return options.artist ? (urlPrefix = "Artists/" + instance.encodeName(options.artist), delete options.artist) : options.person ? (urlPrefix = "Persons/" + instance.encodeName(options.person), delete options.person) : options.genre ? (urlPrefix = "Genres/" + instance.encodeName(options.genre), delete options.genre) : options.musicGenre ? (urlPrefix = "MusicGenres/" + instance.encodeName(options.musicGenre), delete options.musicGenre) : options.studio ? (urlPrefix = "Studios/" + instance.encodeName(options.studio), delete options.studio) : (urlPrefix = "Items/" + options.itemId, delete options.itemId), urlPrefix
```

Comment:
Maybe de-uglify?
```suggestion
        if (options.artist) {
            urlPrefix = "Artists/" + instance.encodeName(options.artist);
            delete options.artist;
        } else if (options.person) {
            urlPrefix = "Persons/" + instance.encodeName(options.person);
            delete options.person;
        } else if (options.genre) {
            urlPrefix = "Genres/" + instance.encodeName(options.genre);
            delete options.genre;
        } else if (options.musicGenre) {
            urlPrefix = "MusicGenres/" + instance.encodeName(options.musicGenre);
            delete options.musicGenre;
        } else if (options.studio) {
            urlPrefix = "Studios/" + instance.encodeName(options.studio);
            delete options.studio;
        } else {
            urlPrefix = "Items/" + options.itemId;
            delete options.itemId;
        }
        return urlPrefix;
```

Reply:
Not sure there's much of a point. We have the real source in the apiclient repo. It just hasn't been decided what to do with it yet.

Reply:
Fair point